pixeltable/tests/test_exprs.py

```diff
@@ -356,6 +358,28 @@ def test_arrays(self, test_tbl: catalog.Table) -> None:
         _ = t[t.array_col[:, 0]].show()
         print(_)
 
+    def test_type_cast(self, test_tbl: catalog.Table) -> None:
+        t = test_tbl
+        # For each column c2, ..., c7, we create a new column ci_as_str that casts it to
+        # a string, then check that each row is correctly converted
+        for col_id in range(2, 8):
+            col_name = f'c{col_id}'
+            as_str_col_name = f'c{col_id}_as_str'
+            t.add_column(**{as_str_col_name: getattr(t, col_name).astype(StringType())})
```

Comment:
status = t.add_column(...)
assert status.num_excs == 0

Reply:
Added, though it's not strictly necessary (the test will fail later if something goes wrong with column creation)